ckanext/unaids/theme/templates/user/edit_user_form.html

```diff
@@ -6,17 +6,28 @@
   <fieldset>
     <legend>{{ _('Change details') }}</legend>
 
-    <p>{{ _('To change your profile settings, please ') }}<a href="{{ h.get_ape_url() }}">{{ _('click here') }}</a>.</p>
+    <label>{{ _('Profile data') }}</label>
```

Comment:
Sorry one final comment I forgot to put into my last review.  Can you just update this to say "HIV Tools Profile Data:"